dap-kotlin.el

```diff
@@ -4,13 +4,25 @@
 (require 'lsp-kotlin)
 (require 'dap-mode)
 
+(defcustom dap-kotlin-enable-log nil
+  "Turn on logging of debug adapter logs to file."
+  :type 'boolean
+  :group 'dap-kotlin)
+
+(defcustom dap-kotlin-log-path nil
+  "Path to the debug adapter log file."
+  :type 'string
+  :group 'dap-kotlin)
+
 (defun dap-kotlin-populate-launch-args (conf)
   ;; we require mainClass to be filled in in the launch configuration.
   ;; dap-kotlin does currently not support a fallback if not defined
   (-> conf
       (dap--put-if-absent :request "launch")
       (dap--put-if-absent :name "Kotlin Launch")
-      (dap--put-if-absent :projectRoot (lsp-workspace-root))))
+      (dap--put-if-absent :projectRoot (lsp-workspace-root))
+      (dap--put-if-absent :enableJsonLogging dap-kotlin-enable-log)
```

Comment:
usually in this case is good to use `lsp-json-bool`

Reply:
Updated the PR 🙂